### PR TITLE
[WIP] Some proposed changes

### PR DESCRIPTION
**Pull request by simonbray**

### CODE REVIEW

File: topics/statistics/tutorials/regression_machinelearning/tutorial.md [hunk 1]
```diff
@@ -262,72 +279,60 @@ To learn the mapping between several features and the targets, in the next step,
...
+After the **New Pipeline/Estimator** and its tunable hyperparameters are produced, we will use the **Hyperparameter search** tool to find the best values for each hyperparameter. These values will lead us to create the best model based on the search space chosen for each hyperparameter. We use only one parameter `n_estimators` of `Gradient boosting` regressor for this task. This parameter specifies the number of boosting stages the learning process has to go through. The default value of `n_estimators` for this regressor is `100`. However, we are not sure if this gives the best accuracy. Therefore, it is important to try setting this parameter to different values to find the optimal one. We choose some values which are less than `100` and a few more than `100`. The hyperparameter search will look for the optimal number of estimators and gives the best-trained model as one of the outputs. This model is used in the next step to predict age in the test dataset.
```
Comment (simonbray) [inline topics/statistics/tutorials/regression_machinelearning/tutorial.md]: @qiagu is New Pipeline/Estimator a tool? It was not mentioned in the tutorial as far as I can see.

**Comment by bgruening** [comment]
@simonbray thanks a lot. There are too many changes in this PR, it that correct?

File: topics/statistics/tutorials/regression_machinelearning/tutorial.md (hunk 1 above)
Comment (qiagu) [reply topics/statistics/tutorials/regression_machinelearning/tutorial.md]: Not exactly, but it's the default output name of the `pipeline builder` tool. Feel free to change to a more proper format if you have one in mind. Thanks.

Comment (simonbray) [reply topics/statistics/tutorials/regression_machinelearning/tutorial.md]: Makes sense, thanks.

**Comment by simonbray** [comment]
Will have another look at this this week, don't merge yet

**Review by khanteymoori — APPROVED** [review]
Thank you so much Simon for your comments and modifications.

**Pull request merged**